molgenis-app/pom.xml

```diff
@@ -52,6 +52,23 @@
 						<artifactId>mysql-connector-java</artifactId>
 						<version>${mysql.connector.java.version}</version>
 					</dependency>
+					<dependency>
+						<groupId>uk.ac.ebi.mydas</groupId>
+						<artifactId>mydas</artifactId>
+						<version>1.6.7</version>
+						<scope>system</scope>
+						<systemPath>${basedir}/../molgenis-das/src/main/resources/mydas-1.6.7.jar</systemPath>
+					</dependency>
+					<dependency>
+						<groupId>xpp3</groupId>
+						<artifactId>xpp3</artifactId>
+						<version>1.1.3.4.O</version>
+					</dependency>
+					<dependency>
```

Comment:
Jetty does not run, stating it is missing the Log4j class dependency
I suspect this is caused due to MyDas being added here